scripts/generateMaintenanceTestData.sh

```diff
@@ -1,22 +1,61 @@
 #!/bin/bash
-# FIXMEMAINTENANCE: please add a couple of lines about why this script exists
+#
+# Generate test data for maintenance unit tests
+#
+
+# collections for system database
+echo "== Creating collection _system/bar =="
+curl [::]:8530/_api/collection -sd \
+     '{"name":"bar", "type": 3, "numberOfShards": 9, "replicationFactor": 2}' \
+    | jq -c
+curl [::]:8530/_api/index?collection=bar -sd \
+     '{"fields":["gi"],"geoJson":false,"sparse":true,"type":"geo","unique":false}' \
+    |jq -c
+curl [::]:8530/_api/index?collection=bar -sd \
+     '{"fields":["gij"],"geoJson":true,"sparse":true,"type":"geo","unique":false}'\
+    |jq -c
+curl [::]:8530/_api/index?collection=bar -sd \
+     '{"deduplicate":false,"fields":["hi","_key"],"sparse":false,"type":"hash","unique":true}'|jq -c
+curl [::]:8530/_api/index?collection=bar -sd \
+     '{"deduplicate":false,"fields":["pi"],"sparse":true,"type":"persistent","unique":false}'|jq -c
+curl [::]:8530/_api/index?collection=bar -sd \
+     '{"fields":["fi"],"id":"2010132","minLength":3,"sparse":true,"type":"fulltext","unique":false}'|jq -c
+curl [::]:8530/_api/index?collection=bar -sd \
+     '{"deduplicate":true,"fields":["sli"],"sparse":false,"type":"skiplist","unique":false}'
+
+echo "== Creating collection _system/baz =="
+curl [::]:8530/_api/collection -sd \
+     '{"name":"baz", "type": 2, "numberOfShards": 1, "replicationFactor": 2}' | jq -c
+# create foo database
+echo "== Creating database foo =="
+curl [::]:8530/_api/database -sd '{"name":"foo"}' | jq -c 
+echo "== Creating collection foo/foobar =="
+curl [::]:8530/_db/foo/_api/collection -sd \
+     '{"name":"foobar", "type": 3, "numberOfShards": 4, "replicationFactor": 3}' \
+    | jq -c 
+echo "== Creating collection foo/foobaz =="
+curl [::]:8530/_db/foo/_api/collection -sd \
+     '{"name":"foobaz", "type": 2, "numberOfShards": 6, "replicationFactor": 2}' \
+    | jq -c
+
+
 
 header="R\"=("
 footer=")=\""
 
 outfile=Plan.json
 echo $header > $outfile
-curl -s localhost:4001/_api/agency/read -d'[["/arango/Plan"]]'|jq .[0].arango.Plan >> $outfile
+curl [::]:4001/_api/agency/read -sLd'[["/arango/Plan"]]'|jq .[0].arango.Plan >> $outfile
```

Comment:
needs to be localhost for ipv4 compatibility

Reply:
we don't need IP4, really!
let freaking ip 4 die. we don't need it. we have tons o scripts, that don't care about that. come on folks, let's not do this kind of pseudo fixes. this is an internal script like startLocalCluster.